src/types/dataModel/volunteer.ts

```diff
@@ -0,0 +1,43 @@
+import { z } from 'zod';
+import zOrganization from './organization';
+
+export const roles = ['Medical', 'Dental', 'Food', 'Save the Babies'] as const;
+export const zRole = z.enum(roles);
+export type Role = z.infer<typeof zRole>;
+
+export const backgroundCheckStatuses = [
+  'Passed',
+  'Failed',
+  'In Progress',
+  'Not Checked',
+] as const;
+export const zBackgroundCheckStatus = z.enum(backgroundCheckStatuses);
+
+const zVolunteer = z.object({
+  firstName: z.string(),
+  lastName: z.string(),
+  email: z.string().email(),
+  phoneNumber: z.string(),
+  previousRole: zRole,
```

Comment:
Should this be optional? What if I haven't volunteered before?

Reply:
true that